.github/workflows/main.yml

```diff
@@ -2,12 +2,12 @@ name: Master Branch CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
```

Comment:
format-on-save made these changes. Is there a standardized style ruleset for this repo to prevent constant trashing like this, @domesticmouse?

Reply:
I'm unaware of a standard format for yaml files.